### DISCUSSION TITLE
request for new features or apps for linux mint 22

### DISCUSSION THREAD

**Discussion by adelpourmand** [linuxmint/discussions#485] [Ideas]
Hello
i would like to request several packages that i believe should be pre-installed on next version of linux mint. To my knowledge, it seems they are necessary:
1- mint needs a miracast software for wireless monitor screencasting (for both audio and video) such as gnome network displays or etc
2- calculate sha256 which must be in action menu or pre-installed sha calculator such as nemo digests ("sudo apt install nemo-gtkhash"). in addition it seems nemo file manager needs a preview pane such as what windows OS has. 
3- preinstalled download manager such as persepolis or FDM inaddition to preinstalled extentions for browsers
4- pre-addeded libreoffice repository instead of manual input of "sudo add-apt-repository ppa:libreoffice/ppa"
5- mint needs a "Print to PDF tool" or applet or software
6- audio tools such as: 1-PulseEffects, 2-PulseAudio
7- mint needs preinstalled password manager such as KeepassXC
8- Mint need a modern system monitor such as Conky manager or mission center
9- Suricata/Snort or any GUI software with the same capabilities that can monitor network connections and etc ...
10- any screen recorder such as simple screen recorder 
11- linux mint needs an preinstalled software to be able to install windows softwares on linux such as wine, playonlinux or etc
12- any preinstalled applet or extension or tool to be able to control gamma, temperature, brightness and contrast of monitor
13- color profile viewer: to be able to understand which color profile has which specifications and effects
14- new screensaver such as Xscreensaver
15- any preinstalled software to be able to execute android apps in linux mint
16- i think use of "pix" is more helpful than that simple "image viewer". pix has several simple editing tools that can quickly edit an image.

Tnx.

**Comment by umikali222** [comment]
I don't get a few of these like the sha256 calculator. If you want to calculate it from a file, you can either do it through the terminal, or I'm pretty sure you can just do that from the properties window of a file.

**Comment by umikali222** [comment]
Also wine shouldn't be installed by default, because some people prefer to only use apps that are available for Linux natively and don't want to have wine installed (not me just some people). Also you can very easily install it from the software manager .

**Comment by enigma131** [comment]
Kodi 21.1 is missing from LM 22, I have open a thread but no response:
https://forums.linuxmint.com/viewtopic.php?p=2542233#p2542233

**Comment by Jopp-gh** [comment]
I know, learning new software is a huge deal breaker.

The Reality is that Linux is neither an emulation of Win nor Mac - I agree however, the default installation isn't too useful for users with a particular set of applications.

From my experience,  default installations **must be boring** in order to be an "useful base" for everyone.
So, defaults are just a start for everyone to build upon.

Wine has nothing to do with Linux and the last thing I'd want to have on my Linux, because windows is the main target for malware.

With Mint, users can decide their own path, commercial OS's are more restrictive. 

Some OS's like Zorin offer various installer options for:
Students, Content creators and more. 

I agree, Mint's main installer should offer various *sets of applications* (office, artistic, coding, etc).
